docs/05-technical-reference/05-configuration-parameters/apix-02-ory-chart.md

```diff
@@ -12,10 +12,33 @@ This table lists the configurable parameters, their descriptions, and default va
 
 | Parameter |  Description | Default value |
 |-------|-------|:--------:|
+| **global.postgresql.postgresqlDatabase** | Specifies the name of the database saved in Hydra. | `db4hydra` |
+| **global.postgresql.postgresqlUsername** | Specifies the username of the database saved in Hydra. | `hydra` |
+| **global.istio.gateway.name** | Specifies the name of the Istio Gateway used in Hydra. | `kyma-gateway` |
+| **global.istio.gateway.namespace** | Specifies the Namespace of the Istio Gateway used in Hydra. | `kyma-system` |
+| **global.ory.oathkeeper.maester.mode** | Specifies the mode in which ORY Oathkeeper Maester is expected to be deployed. | `sidecar` |
+| **global.ory.hydra.persistence.enabled** | Sets persistence for Hydra. | `true`|
+| **global.ory.hydra.persistence.postgresql.enabled** | Defines whether Hydra should initiate the deployment of an in-cluster database. Set to `false` to use a self-provided database. If set to `true`, Hydra always uses an in-cluster database and ignores the custom database details. | `true` |
+| **hpa.oathkeeper.minReplicas** |  Defines the initial number of created Oathkeeper instances. | `1` |
+| **hpa.oathkeeper.maxReplicas** |  Defines the maximum number of created Oathkeeper instances. | `3` |
+| **hydra.deployment.resources.limits.cpu** | Defines limits for CPU resources. | `500m` |
```

Comment:
```suggestion
| **hydra.deployment.resources.limits.cpu** | Defines limits for CPU resources in Hydra. | `500m` |
```

This is just nitpicking, but there are a lot of repeating/identical descriptions in the doc for `Defines limits for CPU resources`, `Defines requests for CPU resources`, etc. Maybe this can be more specific, but then again, maybe it's visible enough from the parameter name what the description is meant for. 🤷